aquarius/events/constants.py

```diff
@@ -2,10 +2,27 @@
 # Copyright 2021 Ocean Protocol Foundation
 # SPDX-License-Identifier: Apache-2.0
 #
-"""
-Defines values of variables:
-1. `EVENT_METADATA_CREATED`
-2. `EVENT_METADATA_UPDATED`.
-"""
-EVENT_METADATA_CREATED = "MetadataCreated"
-EVENT_METADATA_UPDATED = "MetadataUpdated"
+
+from enum import Enum, IntEnum
+
+
+class Events(Enum):
```

Comment:
I would suggest EventTypes or EventClasses. Otherwise from just reading the class name, it seems like it is a collection of event objects.